app/code/Magento/ProductAlert/Model/Email.php

```diff
@@ -378,12 +378,12 @@ public function send()
                 'customerName' => $customerName,
                 'alertGrid' => $alertGrid,
             ]
-        )->setFrom(
+        )->setFromByScope(
             $this->_scopeConfig->getValue(
                 self::XML_PATH_EMAIL_IDENTITY,
                 ScopeInterface::SCOPE_STORE,
                 $storeId
-            )
+            ), $storeId
```

Comment:
Please move this argument declaration to the new line

Reply:
thanks, its done.